### ISSUE TITLE
Plugin request: Open Document format

### ISSUE THREAD

**Issue by tanc** [ccseer/Seer#298]
I'm testing out Seer and noticed immediately that the Open Document format isn't supported. Is it possible to have these added? See https://en.wikipedia.org/wiki/OpenDocument

Thanks!

**Comment by ccseer** [comment]
Did you install OIT plugin?

**Comment by tanc** [comment]
Yes I did, it supports excel, word etc but not the open document standard

**Comment by ccseer** [comment]
Add the extensions of open document formats to the end of Extension field.

Then try again.

**Comment by tanc** [comment]
This works! Thanks

Steps for anyone else wanting to open document format previews:

1. Open the settings of Seer
2. Install the OIT plugin (follow the [docs](http://1218.io/seer/plug-in.html))
3. In the Plugins section of the Seer settings click the "Local" tab
4. You should see the OIT plugin listed. Double click the plugin in the list and it'll open it for editing
5. In the file extensions field, after the last file extension add the extensions you need, each one separated by four spaces. I added `    odt    ods` to the end of mine.
6. Click the tick icon to save the changes
7. Now highlight your odt or ods file and use Seer on it and you should see a preview

So people don't have to do this in the future would it be possible to add open document file extensions to the plugin?

I'll close this as there is a solution already. Thanks for your quick response and help!

**Comment by tanc** [comment]
![image](https://user-images.githubusercontent.com/76320/117108894-22703780-ad84-11eb-849d-8a5a6cef944c.png)

A screenshot of the plugin settings

**Comment by ccseer** [comment]
Maybe next time I need to update the plugin, I will add these  extensions.

**Comment by ccseer** [comment]
Can only Pin  3 issues on the top, so open this one as solved.

**Comment by ccseer** [comment]
With the release of Seer-3.0.0, these file formats are added by default in OIT-1.2.0.
